AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -43,16 +40,57 @@ export var functionsFlow = {
 
         if (!await isKuduAccessiblePromise)
         {
-            if (siteInfo.kind.includes("linux") || siteInfo.kind.includes("container")){

```

Comment:
why are we modifying this logic?

Reply:
While check in my latest code to Git , I have missed the changes from yifguoMSFT (on Jan 11) . I will take the latest code from Main branch and do check in again.